site/content/upgrading-to-v4.md

```diff
@@ -0,0 +1,92 @@
+---
+description: Guide on upgrading to v4 and above
+docs: DOCS-000
+title: Guild on Upgrading to v4 and above
+toc: true
+weight: 2100
+headless: true
+---
+
+{{< note >}}
+If you are using NGINX Ingress Controller version `v3.3.2` (Helm Chart `v1.0.2`) or earlier, please upgrade to `v3.7.2` (Helm Chart `v1.4.2`), and convert your TransportServer and GlobalConfiguration resources to `apiVersion` `k8s.nginx.org/v1`.
```

Comment:
```suggestion
If you are using NGINX Ingress Controller version `v3.3.2` (Helm Chart `v1.0.2`) or earlier, please upgrade to `v3.7.2` (Helm Chart `v1.4.2`), and convert your TransportServer, GlobalConfiguration and Policy resources to `apiVersion` `k8s.nginx.org/v1`.
```

Reply:
Policy was at `apiVersion` `v1` much earlier than this. 4/5+ years ago from Git blame.